setup.cfg

```diff
@@ -16,7 +16,7 @@ line_length=88
 
 [tool:pytest]
 testpaths=tests/
-adopts= --cov --cov-fail-under=20
+addopts= --cov --cov-fail-under=20
```

Comment:
I think this should be left as adopts?

Reply:
Thank you for reviewing. :)
I believe it should indeed be addopts:
https://docs.pytest.org/en/7.1.x/reference/reference.html#confval-addopts
The function "adds options". Besides, it has been addopts already once in the initial commit of the setup.cfg.